anesthetic/utils.py

```diff
@@ -1,10 +1,31 @@
 """Data-processing utility functions."""
 import numpy
 import pandas
+from scipy import special as sp
```

Comment:
I would just use ``import scipy.special`` and avoid the `sp` abbreviation. I'm not sure how standard this is, and it's more in keeping with the rest of `anesthetic`

Reply:
Good point, `sp` is totally unnecessary here. Taken care of in [7b893da](https://github.com/williamjameshandley/anesthetic/pull/71/commits/7b893dab10e1be9410ec4ff814d9ac663455c47d).

On another note, though, I personally would be very much in favour of using `numpy as np`. Have often stumbled over writing `np` first and then having to correct it. I think that `np` is a very widely used abbreviation and this would also shorten the code and make it easier to keep things on one line and hence more readable.

Reply:
I know. It's really just my idiosyncrasy to avoid using `np`. You should raise it as a separate issue to remind us to possibly make this wholesale conversion (which shouldn't be too difficult to do).

Reply:
> You should raise it as a separate issue to remind us to possibly make this wholesale conversion

Done in #76.